models/erfc_neuron.h

```diff
@@ -0,0 +1,142 @@
+/*
+ *  erfc_neuron.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef ERFC_NEURON_H
+#define ERFC_NEURON_H
+
+// Includes from models:
+#include "binary_neuron.h"
+#include "binary_neuron_impl.h"
```

Comment:
Header files should not include `*_impl.h` files. If you need methods defined in and `*_impl.h` file, you need to move that code to `erfc_neuron.cpp`.